README.md

```diff
@@ -1,24 +1,183 @@
+# 📗 Table of Contents
 
-* Configuration
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Full-Stack-Capstone-Backend](#-full-stack-capstone-backend)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [Front-End-Link](#front-end-link)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
 
-* Database creation
+<!-- PROJECT DESCRIPTION -->
 
-* Database initialization
+# 📖 Full-Stack-Capstone-Backend
 
-* How to run the test suite
+**The Full-Stack-Capstone-Backend** is a mobile web application where user can manage the budget.
 
-* Services (job queues, cache servers, search engines, etc.)
+## 🛠 Built With <a name="built-with"></a>
+- ✅ Ruby
+- ✅ Ruby on Rails
+- ✅ Postgresql
+- ✅ Linters
 
-* Deployment instructions
+### Tech Stack <a name="tech-stack"></a>
 
-* ...
+<details>
+  <summary>Language</summary>
+  <ul>
+    <li>Ruby-on-Rails</li>
+    <li>React</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- 🔰 **Splash Welcome page**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Front-End-Link
+
+The front-end repo is available [here](https://github.com/bhushan354/Full-Stack-Capstone-Backend)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+**To get a local copy up and running, follow these steps.**
+
+1. Download or clone this [repostory](git@github.com:bhushan354/Full-Stack-Capstone-Backend.git).
+2. Provide a modern web browser.
+
+### Prerequisites
+
+**In order to run this project you need:**
+
+- ✔ Ruby installed in your machine. you can download it from [here](https://www.ruby-lang.org/en/downloads/)
+- ✔ IDE or a code editor installed in your machine.
+- ✔ IRB.
+- ✔ Get Postgresql up and running. you can download it from [here](https://www.postgresql.org/download/windows/).
+- ✔ [Git](https://git-scm.com/downloads) installed in your machine.
+- ✔ Sign in or sign up to your [Github](https://github.com/) account.
+- ✔ A professional editer such as [VS Code](https://code.visualstudio.com/download).
+- ✔ An Updated web browser such as Google Chrome, you can download it from [here](https://www.google.com/chrome/).
+
+### Setup
+
+- Clone this [repository](git@github.com:bhushan354/Full-Stack-Capstone-Backend.git) to your desired folder:
+
+- Run this command in your command line interface:
+
+```sh
+  cd [YOUR FOLDER]
+  git git@github.com:bhushan354/Full-Stack-Capstone-Backend.git
+  cd Full-Stack-Capstone-Backend
+  rails db:migrate
+  rails db:seed
```

Comment:
- [ ] Before migrating and seeding the database, please include the command `rails db:create` so your users are able to create the database. 👍